packages/graphql-validate-fixtures/src/index.ts

```diff
@@ -1,68 +1,66 @@
 import {readFile, readJSON} from 'fs-extra';
-import {isAbsolute, resolve} from 'path';
-import {
-  GraphQLSchema,
-  buildClientSchema,
-  Source,
-  parse,
-  concatAST,
-} from 'graphql';
+import {resolve} from 'path';
+import * as glob from 'glob';
+import {Source, parse, concatAST, GraphQLSchema} from 'graphql';
+import {getGraphQLConfig, GraphQLProjectConfig} from 'graphql-config';
 import {compile} from 'graphql-tool-utilities/ast';
+import {
+  getGraphQLFilePath,
+  getGraphQLProjects,
+} from 'graphql-tool-utilities/config';
 
 import {
-  validateFixtureAgainstAST,
-  validateFixtureAgainstSchema,
-  Validation,
   Fixture,
+  Validation,
+  validateFixture,
+  getOperationForFixture,
+  MissingOperationError,
+  AmbiguousOperationNameError,
+  GraphQLProjectOperations,
 } from './validate';
 
-export interface Paths {
-  fixturePaths: string[];
-  operationPaths?: string[];
-  schemaPath: string;
-}
-
 export interface Options {
-  schemaOnly?: boolean;
+  cwd: string;
 }
 
 export interface Evaluation extends Validation {
-  fixturePath: string;
   scriptError?: Error;
 }
 
 export async function evaluateFixtures(
-  {fixturePaths, operationPaths = [], schemaPath}: Paths,
-  {schemaOnly = false}: Options = {},
+  fixturePaths: string[],
+  {cwd}: Options,
 ): Promise<Evaluation[]> {
-  let schema: GraphQLSchema;
+  const config = getGraphQLConfig(resolve(cwd));
 
-  try {
-    const schemaJSON = await readJSON(schemaPath, {encoding: 'utf8'});
-    schema = buildClientSchema(schemaJSON.data);
-  } catch (error) {
-    throw new Error(
-      `Error parsing '${schemaPath}':\n\n${error.message.replace(
-        /Syntax Error GraphQL \(.*?\) /,
-        '',
-      )}`,
-    );
-  }
+  const projectOperationCollection = await Promise.all(
+    getGraphQLProjects(config).map(getOperationsForProject),
+  );
 
-  if (schemaOnly) {
-    return runForEachFixture(fixturePaths, (fixture) =>
-      validateFixtureAgainstSchema(fixture, schema),
-    );
-  }
+  return runForEachFixture(fixturePaths, (fixture) =>
+    evaluateFixture(fixture, projectOperationCollection),
+  );
+}
+
+async function getOperationsForProject(
+  projectConfig: GraphQLProjectConfig,
+): Promise<GraphQLProjectOperations> {
+  const operationPaths = projectConfig.includes
+    .map((filePath) => getGraphQLFilePath(projectConfig, filePath))
+    .reduce<string[]>((filePaths, pattern) => {
+      return filePaths.concat(glob.sync(pattern));
+    }, [])
+    .filter((operationPath) => projectConfig.includesFile(operationPath));
```

Comment:
Why do you need this filter?

Reply:
This needs to get replaced with proper globbing like i am doing with #33 
The original intent was to remove excluded files from the list since we were just globbing on all includes.